Create lwm2m-agent-1021-11-28-updated-clients-sdk-to-version-containing-n-2-token-fix-graft.md (#4502)
SoftwareAG/c8y-docs: content/change-logs/device-management/lwm2m-agent-1021-11-28-updated-clients-sdk-to-version-containing-n-2-token-fix-graft.md

---
date: ""

> title: Fixed Lwm2m operation execution delay

Reviewer: ```suggestion
title: Fixed LWM2M operation execution delay
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:
  - value: tc-ggH2M4hf3
    label: lwm2m-agent
ticket: DM-5841
version: 1021.11.28
---

> Fixed an issue where Lwm2m agent failed to execute operations immediately after their creation.

Reviewer: ```suggestion
The LWM2M agent failed to execute operations immediately after their creation. This issue has been fixed.

> Fixed an issue where the agent would fail to refresh JWT tokens following periods of inactivity, preventing automatic recovery and requiring a manual service restart.

Reviewer 2: The title and the description it's not appropriate to user documentation.

"requiring a manual service restart" -> user can't do that.
What they will experience is the actual thing to be written here. 
Is it operation could not be delivered in realtime and caused delays as they were taken later when device connects again?

Reviewer 3: done

---
date: ""

> title: Fixed an issue with notifications token refresh in LWM2M agent

Reviewer: Better also change to "JWT"?

Reviewer 3: I completely modified this change log entry as suggested by @Oezge-Akin

ticket: DM-5841
version: 1021.11.28
---

> Fixed an issue where the agent would fail to refresh JWT tokens following periods of inactivity, preventing automatic recovery and requiring a manual service restart.

Reviewer: ```suggestion
The LWM2M agent previously failed to refresh JWT tokens following periods of inactivity, preventing automatic recovery and requiring a manual service restart. This issue has been fixed.
```

Reviewer 3: I completely modified this change log entry as suggested by @Oezge-Akin
